test/CMakeLists.txt

```diff
@@ -136,3 +136,11 @@ itk_add_test(NAME itkPhaseCorrelationImageRegistrationMethodTestTiles
     ${CMAKE_CURRENT_LIST_DIR}/Input/Tiles
     ${TESTING_OUTPUT_PATH}/itkMockMontageTestTiles
   )
+
```

Comment:
For now, please add a check to see if the `Input/NoisyTiles` directory exists before adding this test to avoid [false positives](https://open.cdash.org/testDetails.php?test=645843033&build=5343315).

The other tests could be fixed up so we have green CI testing.